### DISCUSSION TITLE
[Idea]: Mute Zoom audio output when background music is playing

### DISCUSSION THREAD

**Discussion by baciucristian** [sircharlo/meeting-media-manager#1559] [Ideas] [answered]
### Is your idea related to a problem?

Every time we need to mute the Zoom Audio Output from the Volume Mixer when we are playing the background music, and to turn it on when the meeting starts. We need to do the same thing after the meeting ends. Sometimes we forgot to do that, and because of that when someone are commenting, we can't hear anything in the KH.
<img src="https://user-images.githubusercontent.com/49842769/220932997-697ae633-edec-4a9f-9dd5-571a8849eff6.png" width=50%>

### Describe your idea

It would be nice if when the background music is playing, to mute automatically the Zoom audio, without going manually to the volume mixer setting. And to turn it on when the background music stops playing (when the meeting starts).

### Describe alternatives you've considered

Every time we need to mute manually the Zoom Audio Output from the Volume Mixer (Windows 10) when we are playing the background music.

**Comment by sircharlo** [comment, marked as answer]
Related to ongoing discussion here #1303 
In our KH, we send the AV Zoom connection to a seperate breakout room before the meeting to get around this problem without playing with the volume settings.
Putting the AV Zoom connection in the waiting room until right before the meeting would also have the intended effect.

**Comment by mtdvlpr** [comment]
@baciucristian, what we do is not connect to the Zoom audio until 1 minute before the meeting and afterwards, in Zoom, "disconnect from computer audio". This way the KH doesn't hear the Zoom participants and the Zoom participants don't hear the KH (can talk to each other)

**Reply by baciucristian** [reply]
Great idea, I didn't thought of that. It easier than messing with the Volume Mixer. It would be nice a way to automatize that in the M3 tho 🙂

**Reply by mtdvlpr** [reply]
Zoom integration is in the test phase (see #1303). Automating the audio is unfortunately not possible, but we are working on a workflow where the same outcome could be achieved.
